---
title: '&#39;Catch&#39; bloco nunca alcançado; &lt;exceção&gt; manipulado acima na mesma &#39;tente&#39; instrução'
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: f4cebeb968d6a790d2c365c9b1c151f80794eb7f
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 05/04/2018
---
# <a name="39catch39-block-never-reached-ltexceptiongt-handled-above-in-the-same-39try39-statement"></a>&#39;Catch&#39; bloco nunca alcançado; &lt;exceção&gt; manipulado acima na mesma &#39;tente&#39; instrução
Um `Catch` bloco no código não pode ser alcançado porque ela é tratada em precedidos `Try` bloco.  
  
 Por padrão, esta mensagem é um aviso. Para obter mais informações sobre como ocultar avisos ou tratar avisos como erros, consulte [Configurando avisos no Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)  
  
 **ID do erro:** BC42031  
  
## <a name="to-correct-this-error"></a>Para corrigir este erro  
  
-   Remova a instrução redundante.  
  
## <a name="see-also"></a>Consulte também  
 [Como: capturar uma exceção no Visual Basic](http://msdn.microsoft.com/library/f3063c89-d2bf-49b1-91b5-b87edfb18b95)  
 [Como: testar o código com um bloco Try... Bloco Catch no Visual Basic](http://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [Como: Filtrar erros em um bloco Catch no Visual Basic](http://msdn.microsoft.com/library/85964d0a-56e7-4301-a96e-5eaea23b7b9b)  
 [Passo a passo: Tratamento estruturado de exceções (Visual Basic)](http://msdn.microsoft.com/library/440da655-4b32-490b-8b16-bfe46f41fa76)  
 [Instrução Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
